test/transactionBreakdown.test.ts

```diff
@@ -58,30 +58,96 @@ describe('transactionBreakdown routes', () => {
       ).toBe(true);
     });
 
-    it('should throw an error if any field (tax, tip, party and/or items) are omitted', async () => {
-      let res = await request
+    it('should return a transactionBreakdown with 0 tax', async () => {
+      const res = await request
+        .post('/api/transactionBreakdown')
+        .send(transactionBreakdownSeed.transactionBreakdownZeroTax);
+      const { orderTransaction, orderTransactionPersons } = res.body;
+
+      expect(res.status).toBe(200);
+      expect(
+        _.isEqual(
+          orderTransaction,
+          transactionBreakdownSeed.transactionZeroTaxComparison.orderTransaction
+        )
+      ).toBe(true);
+      expect(
+        _.isEqual(
```

Comment:
nit: i think _.isEqual is already an assertion so you don't need to wrap it around `expect(...).toBe(true)`